TowerDefense/StaticClasses/GameStats.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+namespace TowerDefense
+{
+    static class GameStats
+    {
```

Comment:
I don't know if this is the best place for this stuff, but this stuff needs kind of global access given where I am now, so this works for now.